pkg/server/podsubresourceproxy_handler.go

```diff
@@ -123,7 +125,7 @@ func (b *podSubresourceProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.
 		return
 	}
 
-	// Check if the subresource is valid, we only support exec, log, portforward, proxy and attach.
+	// Check if the subresource is valid, for PODs we support exec, log, portforward, proxy , attach, ephemeralcontainers.
 	if subresource != "exec" && subresource != "log" && subresource != "portforward" && subresource != "proxy" && subresource != "attach" {
```

Comment:
Where is the `ephemeralcontainers` in the `if` ?

Reply:
done